skins/bt.scss

```
@@ -0,0 +1,9 @@
+.yearn {
```

The css selector should similar to the skin name.
```suggestion
.bt {
```

---

You could also remove skin.scss and set yearn for skin in your space settings.

---

> Your space name directory should be similar to your ENS. Follow this example `spaces/my-space.eth/logo.png`.

ok,Has been renamed